app/src/modules/settings/routes/flows/flow.vue

```diff
@@ -76,17 +76,17 @@
 				:hovered-panel="hoveredPanelID"
 				:subdued="flow.status === 'inactive'"
 			/>
-			<v-workspace :panels="panels" :edit-mode="editMode">
-				<template #panel="{ panel }">
+			<v-workspace :tiles="panels" :edit-mode="editMode">
+				<template #tile="{ tile }">
 					<operation
 						v-if="flow"
```

Comment:
Is there a need to propagate the rename of panels into tiles for flows?
The `panels` is passed in as `tiles`. 🤔 Does `panel == tile`?

Reply:
Ref https://github.com/directus/directus/pull/12522#discussion_r881584338 (and the relevant commit https://github.com/directus/directus/pull/12522/commits/a73f8a7fdeefa0e0f60e5a93e26cba7bc25e5ef9)

Good question. Though I assume from the initial point sparking this change (assuming from extensions perspective), maybe there isn't a need to rename as developers will refer them as Flow operations  instead of panels. 🤔

Reply:
> Is there a need to propagate the rename of panels into tiles for flows?

There's not really a need, though it would be a little cleaner. The "workspace" renders "tiles". In the case of Insights, each "Panel" gets converted to a tile and then rendered. In Flows, each "Operation" gets rendered as a "Tile". The name "Panel" is technically used for what we consider the graph/metric/widget/thing in a dashboard